processes/third_party_vuln_process.md

```diff
@@ -89,6 +89,8 @@ Vulnerabilities found and fixed outside this process should be added into
 the vulnerability database. This can be done by anyone through a Pull Request on
 this repository.
 
+The vulnerability should include any kind of supporting material such as references, maintainer review or otherwise to confirm the vulnerability report is valid.
```

Comment:
That is pretty wide. I assume we'll debate in PRs each time if a vuln is valid or not.

Reply:
Agree, but it's probably also tough in advance to list all of the "whitelisted" trusted resources for which you'd "trust" so probably vulns will follow discussions in PR but this is more of a note for triage team members to know not to just merge PRs from external process.

LMK if you have suggestions to further rephrase